### PR TITLE
Made Video Listing page dynamic and embedded vedios

### PR DESCRIPTION
**Pull request by Asuraking1n**
Made Video Listing dynamic.
embedded videos

### CODE REVIEW

File: src/components/cards/vedioCards/VedioCard.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import React,{useState} from 'react'
 import './vediocard.css'
 import Modal from '../../modal/Modal'
-const VedioCard = () => {
+const VedioCard = (props) => {
   const [isModal, setIsModal] = useState(false)
   return (
-    <>{isModal?<Modal openModal={(isModal)=>setIsModal(isModal)}/>:null}
-      <div className="vedio-card-cont-sec" style={{background:"url(https://i.pinimg.com/originals/62/3a/a8/623aa8f9933ee9a286871bf6e0782538.jpg) no-repeat center center/cover"}}>
+    <>{isModal?<Modal openModal={(isModal)=>setIsModal(isModal)} id={props.vedioData._id}/>:null}
```
Comment (arulk8) [inline src/components/cards/vedioCards/VedioCard.js:7]: can we use `isModal && <Modal/>`.

File: src/components/cards/vedioCards/VedioCard.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import React,{useState} from 'react'
 import './vediocard.css'
 import Modal from '../../modal/Modal'
-const VedioCard = () => {
+const VedioCard = (props) => {
```
Comment (arulk8) [inline src/components/cards/vedioCards/VedioCard.js:4]: `const VedioCard = ({_id, vedioData: {title} }) => {` -> object destructuring

File: src/components/modal/Modal.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import Cancel from '../../images/cancel.png'
 import './modal.css'
+import ReactPlayer from 'react-player'
 const Modal = (props) => {
```
Comment (arulk8) [inline src/components/modal/Modal.js:5]: object destructuring can be done for props here as well

File: src/components/modal/Modal.js [hunk 4]
```diff
@@ -9,6 +10,8 @@ const Modal = (props) => {
             <div className="modal-close">
                 <img src={Cancel} alt="cancel" onClick={()=>props.openModal(false)} />
             </div>
+            <ReactPlayer url={`https://www.youtube.com/watch?v=${props.id}`} controls={true} width='100%'
```
Comment (arulk8) [inline src/components/modal/Modal.js:13]: `const YOUTUBE_URL = https://www.youtube.com/watch?v=`

`<ReactPlayer url={`${YOUTUBE_URL}${props.id}}`

File: src/pages/vedioListing/VedioListing.js [hunk 5]
```diff
@@ -1,25 +1,35 @@
...
+          <span>Fashion</span>
+          <span>Beauty</span>
+          <span>Clothing</span>
+          <span>Fashion Shows</span>
+        </div>
+        <div className="vedio-cards-cont" >
+          {!vedio ? <>
+            <img src="https://freefrontend.com/assets/img/css-loaders/loading.gif" alt="load" className='loadImg' />
+          </> :
+            vedio.map((val, index) => {
```
Comment (arulk8) [inline src/pages/vedioListing/VedioListing.js:24]: ` vedio.map((val, index) => <VedioCard key={index} vedioData={val} />)`

**Review by arulk8 — APPROVED** [review]

**Review by sakshi006 — APPROVED** [review]
I liked the code structure and naming conventions.
Make the changes as suggested by @arulk8 [Arulmurugan](https://github.com/arulk8)
and its good to go.

I also like the smooth transition effect in video cards on hover as well.
Also, I am not sure if it is done on purpose but the names should be "video" instead of "vedio"?

**Comment by Asuraking1n** [comment]
> I liked the code structure and naming conventions. Make the changes as suggested by @arulk8 [Arulmurugan](https://github.com/arulk8) and its good to go.
> 
> I also like the smooth transition effect in video cards on hover as well. Also, I am not sure if it is done on purpose but the names should be "video" instead of "vedio"?

It's a typo I will fix it, thanks for pointing out

File: src/components/modal/Modal.js (hunk 3 above)
Comment (Asuraking1n) [reply src/components/modal/Modal.js:5]: Thanks I will fix these...

**Pull request merged**